css-shapes-2/Overview.bs

```diff
@@ -185,19 +210,21 @@ The ''shape()'' Function</h4>
 			Percentages are resolved against the width or height, respectively,
 			of the [=reference box=].
 
-		<dt><dfn><<by-to>></dfn> = <dfn value for="shape(), <by-to>">by</dfn> | <dfn value for="shape(), <by-to>">to</dfn>
+		<dt><dfn><<command-end-point>></dfn> = [ to <<position>> | by <<coordinate-pair>> ]
 		<dd>
 			Every command can be specified in "absolute" or "relative" coordinates,
-			determined by their <<by-to>> component.
-			''to'' indicates that any <<coordinate-pair>>s in the command
+			determined by their <css>by</css> or <css>to</css> component.
+			<css>to</css> indicates that any <<coordinate-pair>>s in the command
 			are relative to the top-left corner of the [=reference box=],
-			while ''by'' indicates that the <<coordinate-pair>>s
+			while <css>by</css> indicates that the <<coordinate-pair>>s
 			are relative to the command's starting point.
```

Comment:
This is no longer true when `from start` / `from end` are used. There probably needs to be a new paragraph that explains how control points are resolved, or a "see below for details".

Reply:
Right... There is a paragraph for control points, I'll refer to that.

Reply:
I think you can use `''shape()/by''` for the markup. I think.

Reply:
Fixed